index.bs

```diff
@@ -2326,10 +2345,6 @@ This method has no arguments and returns a Boolean value.
     };
 </xmp>
 
-Note: Invoking this method from a [=browsing context=] where the [=Web Authentication API=] is "disabled" according to the [=allowed to use=] algorithm&mdash;i.e., by a [=permissions policy=]&mdash;will result in the promise being rejected with a {{DOMException}} whose name is "{{NotAllowedError}}". See also [[#sctn-permissions-policy]].
-
-</div>
-
```

Comment:
Got a bit too much deleted in commit [0cd8e11b](https://github.com/w3c/webauthn/pull/1668/commits/0cd8e11b2edab596645c839afcd3231905aabed5#diff-5e793325cd2bfc452e268a4aa2f02b4024dd9584bd1db3c2595f61f1ecf7b985L2348-L2351) :)